.env.example

```diff
@@ -1,10 +1,20 @@
+VERSION=0.0.1
+VERSION_TAG=latest
+
 DATABASE_ADDRESS=localhost
 DATABASE_USER=postgres
 DATABASE_PASSWORD=postgres
 DATABASE_PORT=5432
 DATABASE_NAME=postgres
 
+WORKERS_AMOUNT=5
+DRIVER=docker
+WORKER_ADDRESS=tcp://localhost:5555
+
+# if ALLOW_ALL is true, an allowed_list file can be provided and it path passed in ALLOWED_LIST_PATH
```

Comment:
isn't it the opposite? If ALLOW_ALL is false we need an allow_list, and it `must` be provided

Reply:
Yep, hahah

Reply:
Addressed it.